mlflow/llama_index/__init__.py

```diff
@@ -332,3 +336,40 @@ def _load_pyfunc(path, model_config: Optional[Dict[str, Any]] = None):
     flavor_conf = _get_flavor_configuration(model_path=path, flavor_name=FLAVOR_NAME)
     engine_type = flavor_conf.pop("engine_type")
     return create_engine_wrapper(_load_index(path, flavor_conf), engine_type, model_config)
+
+
+@experimental
+def autolog(
+    disable: bool = False,
```

Comment:
Nit: should we modify this to be `log_traces`? 

If a user has code running with `autolog(log_traces=True)`, as MLflow updates the functionality the user code functionality will not change. With `disable`, behavior will slowly change over time

Reply:
+1 let's keep a consistent interface so that when we have support for the remaining items (log_models, log_model_signatures, log_input_examples) we can retain that common interface.

Reply:
makes sense!